client/blocks/inline-help/inline-help-search-results.jsx

```diff
@@ -105,31 +106,34 @@ function HelpSearchResults( {
 		}
 	}, [ isSearching, hasAPIResults, searchQuery ] );
 
+	const { setShowSupportDoc } = useDataStoreDispatch( 'automattic/help-center' );
```

Comment:
@alshakero, would love your eyes on this to make sure this is the current way of reaching the Help Center's actions 🙇 Cause I've also seen https://github.com/Automattic/wp-calypso/blob/67f0ccce140ec790c66ca722d7abdb5db3d3ed14/client/components/inline-support-link/index.jsx#L149, and wasn't sure what's the difference 🤔